go.mod

```diff
@@ -0,0 +1,178 @@
+module helm.sh/helm
```

Comment:
I'm still trying to wrap my head around some of the module concepts, especially versioning rules.

Wouldn't this have to be `module helm.sh/helm/v3`?

See:
https://github.com/golang/go/wiki/Modules#semantic-import-versioning
https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher

Reply:
yes, the suffix /v3 needs to be added to the module name. And all the import paths need to refer to the new module name (with suffix /v3). If these two steps are avoided, then this module will be referred to as helm.sh/helm v3.0.0+incompatible

Note - if you are using non-module aware static analyzer tools, then they will fail because such tools will expect v3 folder on the filesystem under GOPATH.

Reply:
does this still need addressing?